packages/mdc-icon-button/_mixins.scss

```diff
@@ -113,7 +115,7 @@
   $feat-color: mdc-feature-create-target($query, color);
   $feat-structure: mdc-feature-create-target($query, structure);
 
-  @include mdc-icon-button-size($mdc-icon-button-size, $query: $query);
+  @include mdc-icon-button-size($mdc-icon-size, $query: $query);
```

Comment:
we can replace this call with density mixin.

Reply:
Got it!